sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -150,7 +150,8 @@ target_link_libraries(azure-core-global-context-test PRIVATE azure-core gtest_ma
 gtest_discover_tests(azure-core-test
      TEST_PREFIX azure-core.
      NO_PRETTY_TYPES
-     NO_PRETTY_VALUES)
+     NO_PRETTY_VALUES
+     DISCOVERY_TIMEOUT 600 )
```

Comment:
Nit: in many cmakefiles, there is an extra space: `600 )`. I suggest to remove space (`600)`), or to move `)` to the next line.

Reply:
@antkmsft done